app/models/pageflow/page.rb

```diff
@@ -16,6 +16,18 @@ def title
       configuration['title'].presence || configuration['additional_title']
     end
 
+    def social_share_url(*args)
+      thumbnail_file.thumbnail_url(*args) unless thumbnail_file.blank?
+    end
+
+    def thumbnail_url(*args)
+      thumbnail_file.thumbnail_url(*args)
+    end
+
+    def thumbnail_file
+      ThumbnailFileResolver.new(page_type.thumbnail_candidates, configuration).find
+    end
+
```

Comment:
`ThumbnailFileResolver` now requires an entry. The `thumbnail_file` method has been moved. You can talk to Daniel about the changes he made in that area.